source/daq/frequency_mask_trigger.hh

```diff
@@ -143,13 +161,49 @@ namespace psyllid
 
         private:
             std::vector< double > f_mask;
-            std::vector< double > f_mask_data;
+            std::vector< double > f_mask2;
+            std::vector< double > f_average_data;
+            std::vector< double > f_variance_data;
             unsigned f_n_summed;
 
             std::mutex f_mask_mutex;
 
     };
 
+    inline void frequency_mask_trigger::calulcate_sigma_mask_spline_points(std::vector< double >& t_x_vals, std::vector< double >& t_y_vals, const double& threshold)
```

Comment:
@nsoblath what do you think about having these as inline? Christine was concerned about having them as a function call so nested, but I don't know if it is worth the cost of putting this sort of implementation in the header... (that is, I don't have a good intuition here so be sure to look at this part)

Reply:
the inlines have been moved into source (also `calulcate`->`calculate`